quantities/units/electromagnetism.py

```diff
@@ -137,6 +137,11 @@
     nF/1000,
     symbol='pF'
 )
+fF = UnitQuantity(
+    'femtofarad',
+    pf/1000,
```

Comment:
this should be `pF/1000`

Reply:
Thank you for spotting that! For some reason, I put nano instead of femto.

Reply:
There is a typo in your code. `pf/1000` will not work, no? So, it should be `pF/1000`.